src/components/CheckboxInput/CheckboxInput.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { Meta, StoryFn } from "@storybook/react";
+import CheckboxInput from "./CheckboxInput";
+import { FormProvider, useForm } from "react-hook-form";
+type FormData = {
+  agreeToTerms: boolean;
+};
+export default {
+  title: "Components/CheckboxInput",
+  component: CheckboxInput,
+} as Meta;
+
+export const Default: StoryFn = () => {
+  const methods = useForm<FormData>();
+
+  // Define a form method to handle form submission (if needed)
+  const onSubmit = (data: FormData) => {
+    console.log("Form submitted with data:", data);
+  };
+
+  return (
+    <FormProvider {...methods}>
+      <form onSubmit={methods.handleSubmit(onSubmit)}>
+        <CheckboxInput
+          name="agreeToTerms"
+          label="I agree to receive automated calls or text messages from Expert Event Solutions LLC in response to my inquiry. Msg & data rates may apply."
+        />
+        <button type="submit">Submit</button>
```

Comment:
On storybook when looking at the checkbox input the text submit shows up at the bottom, without the actual styling for the submit button, I'm not sure if that is an issue or not 
<img width="1080" alt="Screenshot 2024-09-15 at 12 18 09 PM" src="https://github.com/user-attachments/assets/55864144-25af-4973-bc6f-98d084148787">

Reply:
I did add a submit button to the story as well with no styling do you think I should add it with styling?